src/main/app/src/pages/HelpPage.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect } from 'react';
+import '../css/home.css';
+
+// Import Bootstrap CSS and JS. Only this page uses these. 
+import 'bootstrap/dist/css/bootstrap.css';
+import 'bootstrap/dist/js/bootstrap.js';
+
+// This is a stateless, functional React component used to render each resource in a card format. 
+const ResourceCard = ({ organization, description, url }) => (
+  <div className = "p-3">
+    <div className="card">
+      <div className="card-body">
+        <h5 className="card-title text-info">{organization}</h5>
+        <p className="card-text text-secondary" >{description}</p>
+        <a className="card-text btn btn-light" href={url}>View more</a>
+      </div>
+    </div>
+  </div>
+);
+
+/* Component for home page */
+const HelpPage = () => {
+
+  const [resources, updateResources] = React.useState([]);
+
+  useEffect(() => {
+    fetch('https://api.airtable.com/v0/appun5As9N6CFqu4H/Resources?api_key=keyvEcGAXuzDAVmVb')
```

Comment:
You also want to do a check if the resources have already been fetched because as it is right now, the api will keep getting called and resources will keep getting updated infinitely. You could probably add this to component state.

Reply:
So, looks like useEffect() does it for us. I did not use classes, rather I used a const to render the page.  
"What does useEffect do? By using this Hook, you tell React that your component needs to do something after render. React will remember the function you passed (we’ll refer to it as our “effect”), and call it later after performing the DOM updates. In this effect, we set the document title, but we could also perform data fetching or call some other imperative API"